### DISCUSSION TITLE
Profile Repo doesn’t show everything on mobile

### DISCUSSION THREAD

**Discussion by BenjaminMichaelis** [community/community#3428] [Mobile] [answered]
On the mobile profile view, the personal readme, one line is missing on the bottom of the readme even after you click show more. This is on GitHub mobile on the iPhone X.

Pictures show the profile view vs the actual readme.
<img alt="Problem Image" src="https://user-images.githubusercontent.com/22186029/109190623-1e9fb380-774a-11eb-997f-e888c2715601.png" width="400px"/></a>

<img alt="What it should be Image" src="https://user-images.githubusercontent.com/22186029/109190643-25c6c180-774a-11eb-92c8-51653c37dc6f.png" width="400px"/></a>

**Comment by candyho** [comment, marked as answer]
Hey, thanks for flagging this. This is a known bug and it will be fixed in the next our next production release in mid-March. Sorry for the inconvenience caused!

**Comment by Chrome135** [comment]
Ok
